Packet++/src/IPv6Extensions.cpp

```diff
@@ -174,7 +174,7 @@ IPv6TLVOptionHeader::IPv6TLVOptionHeader(const std::vector<IPv6TLVOptionBuilder>
 	for (std::vector<IPv6TLVOptionBuilder>::const_iterator iter = options.begin(); iter != options.end(); iter++)
 	{
 		IPv6Option option = iter->build();
-		memcpy((uint8_t*)(getDataPtr() + offset), option.getRecordBasePtr(), option.getTotalSize());
+		memcpy(getDataPtr() + offset, option.getRecordBasePtr(), option.getTotalSize());
```

Comment:
why did you remove the casting?

Reply:
It is unnecessary because `getDataPtr()` returns `uint8_t*`

Reply:
And `memcpy` gets a `void*` in the first parameter

Reply:
ok